tensortrade/exchanges/simulated/fbm_exchange.py

```diff
@@ -75,8 +86,12 @@ def _generate_price_history(self):
         volume_frame.set_index('date')
         volume_frame.index = pd.to_datetime(volume_frame.index, unit='m', origin=start_date)
 
-        data_frame = price_frame['price'].resample(self._timeframe).ohlc()
-        data_frame['volume'] = volume_frame['volume'].resample(self._timeframe).sum()
+        if self._timeframe != '1min':
```

Comment:
Is this necessary? Won't `pandas` realize the timeframe is already sampled correctly and just return it?